day6/cpp/Capitalise.cpp

```diff
@@ -0,0 +1,22 @@
+/**
+ * @author Bhanu0202
+ * @date 27/12/2018
+ */
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	// your code goes here
+	string sa;
+	getline(cin, sa);
+	int l = sa.length();
+	sa[0] -= 32;
+	for(int i = 0; i < l; i++){
+	    if(sa[i] == 32 && sa[i+1] != 32){
+	        sa[i + 1] -= 32;  
```

Comment:
before subtracting 32 please check if `sa[i + 1]` is a lower case alphabet or not. If it is an upper case letter, subtracting 32 will give incorrect answer.

`Input :`**Abc**
`Output :`**!bc**

Reply:
Thanks for the correction. I will update it soon.